stl/inc/ranges

```diff
@@ -1433,6 +1463,94 @@ namespace ranges {
         inline constexpr _Istream_fn<_Ty> istream;
     } // namespace views
 
+#if _HAS_CXX23
+    namespace _Pipe {
+        template <class _Ty>
+        void _Derived_from_range_adaptor_closure_impl(const volatile range_adaptor_closure<_Ty>&);
+
+        template <class _Ty>
+        concept _Derived_from_range_adaptor_closure = requires(const volatile _Ty& __obj) {

```

Comment:
```suggestion
        concept _Derived_from_range_adaptor_closure = requires(_Ty& _Obj) {
```

We should instead `remove_cvref_t` down below so that we don't need to concern ourselves with cvref-qualifiers at all (same with `_Derived_from_range_adaptor_closure`)